langchain/document_loaders/odt.py

```diff
@@ -0,0 +1,28 @@
+"""Loader that loads Open Office ODT files."""
+from typing import Any, List
+
+from langchain.document_loaders.unstructured import (
+    UnstructuredFileLoader,
+    satisfies_min_unstructured_version,
+)
+
+
+class UnstructuredODTLoader(UnstructuredFileLoader):
+    """Loader that uses unstructured to load open office ODT files."""
+
+    def __init__(
+        self, file_path: str, mode: str = "single", **unstructured_kwargs: Any
+    ):
+        min_unstructured_version = "0.6.3"
+        if not satisfies_min_unstructured_version(min_unstructured_version):
+            raise ValueError(
```

Comment:
There are a few cases (like [here](https://github.com/hwchase17/langchain/blob/master/langchain/document_loaders/unstructured.py#L45-L47)) where we use `satisfies_min_unstructured_version` in conditional logic for backward compatibility, but we could add a `validate_unstructured_version` function to avoid repetition on these.